nano/node/wallet.cpp

```diff
@@ -1507,7 +1566,10 @@ nano::store_iterator<nano::uint256_union, nano::wallet_value> nano::wallet_store
 {
 	return nano::store_iterator<nano::uint256_union, nano::wallet_value> (nullptr);
 }
-
+nano::mdb_wallets_store::mdb_wallets_store (bool & error_a, boost::filesystem::path const & path_a, int lmdb_max_dbs) :
+environment (error_a, path_a, lmdb_max_dbs, 1ULL * 1024 * 1024 * 1024)
```

Comment:
The same GB logic is referenced here as in the mdb_env() constructor in lmdb.cpp. I wonder if it would be beneficial to add our own user defined literal? Then it literally could be written as: 1GB. There is Boost.Units but it only has stuff like length, weight etc.. which is a bit annoying. It's probably a bit out of scope to do it here, but if you are feeling adventurous could always add our own! :)